src/systemd/sd-lldp.h

```diff
@@ -40,6 +41,12 @@ typedef enum LLDPPortStatus {
         _LLDP_PORT_STATUS_INVALID = -1,
 } LLDPPortStatus;
 
+typedef enum LLDPDestinationType {
+        LLDP_DEST_TYPE_NEAREST_BRIDGE,
+        LLDP_DEST_TYPE_NEAREST_NON_TPMR_BRIDGE,
+        LLDP_DEST_TYPE_NEAREST_CUSTOMER_BRIDGE,
+} LLDPDestinationType;
```

Comment:
This _must_ be namespaced properly. Please prefix with `SD_LLDP_DESTINATION_*`. Also please avoid abbreviations.

Note that I just pushed some cleanup-patches to -git which fix the other name-spacing issues.